README.md

```diff
@@ -196,6 +196,25 @@ python3 serve_google_api.py --endpoint localhost:50051 --checkpoint ...
 python3 scripts/stt_google.py --endpoint localhost:50051 --lang ru --api-key-credentials= -i calls_micro/calls_micro.json
 ```
 
+# Frontend performance metrics
+```shell
+bash scripts/read_audio_performance.sh
+```
+| file       | reads count |  backend | process_time us| perf_counter us| 
+|-----------:|---:|---------:|--------------:|-------------:|
+|data/tests/test_5s.wav|  100|       sox|   431341|    11886|
+|data/tests/test_1m.wav|  100|       sox|   455051|    12593|
+|data/tests/test_1h.wav|  100|       sox|  8939791|   458676|
+|data/tests/test_5s.wav|  100|    ffmpeg|  5147064|   140222|
+|data/tests/test_1m.wav|  100|    ffmpeg|  3941069|   306300|
+|data/tests/test_1h.wav|  100|    ffmpeg| 10509560|   628091|
+|data/tests/test_5s.wav|  100| soundfile|    42835|     1680|
+|data/tests/test_1m.wav|  100| soundfile|    36295|     1006|
+|data/tests/test_1h.wav|  100| soundfile|  4311895|   215836|
+|data/tests/test_5s.wav|  100|     scipy|    30163|     1583|
```

Comment:
This still looks off. How come process_time is so much larger (30x) than perf_counter results? process_time does not include wait time and should be smaller. 



I'm getting ~30usec when measuring only `scipy.io.wavfile.read`:
```python
# ffmpeg -f lavfi -i "sine=frequency=1000:duration=5" -c:a pcm_s16le -ar 8000 test.wav

import scipy.io.wavfile
import time

def measure(k, f, audio_path, number = 100, scale = 1000, timer = time.process_time, **kwargs):
    #tic = timer()
    start_process_time = time.process_time_ns()
    start_perf_counter = time.perf_counter_ns()
    for i in range(number):
        audio = f(audio_path, **kwargs)
    #print(k, (timer() - tic) * 1e6 / K, 'microsec')

    end_process_time = time.process_time_ns()
    end_perf_counter = time.perf_counter_ns()
    process_time = (end_process_time - start_process_time) / scale / number
    perf_counter = (end_perf_counter - start_perf_counter) / scale / number
    print(process_time, 'us', perf_counter, 'us')
    return audio

measure('scipy.io.wavfile.read', scipy.io.wavfile.read, 'test.wav')
# 25.86772 us 25.881770000000003 us
```

Here both measures are much closer to each other.

Could you please investigate why is that? Could you please temporarily comment out s2f call?